### ISSUE TITLE
Make a region blacklist

### ISSUE THREAD

**Issue by mrcoffee1026** [kangarko/Boss#71]
Like so that a world can be selected for spawn percentage... but then say a certain selection of regions to NOT spawn in (without selecting any other regions since that would require a region that encompassed the entire rest of the world and exclude a handful of regions nested within... which I don't think can be done anyway)..

**Comment by kangarko** [comment]
Hello,

I am sorry to disappoint but I am currently not planning to change the system in such way.

**Comment by mrcoffee1026** [comment]
Well... I tried to make due with what does exist.   I made a region that encapsulates the world... and of course that includes many other regions where I do NOT want the boss spawned.  However, even though boss spawning is explicitly disabled in these other regions, and even though the bounding region had a priority of -1 (I set to that... thinking maybe it would give the other region setting priority.   Of course it didn't... and the first bosses (killer bunnies) I found were right in spawn.  Even if you weren't planning on making a blacklist feature, it seems to me that it should still obey the "disabled" region setting even if that region exists inside a larger region where the spawning is "enabled".
